docs/conf.py

```diff
@@ -169,17 +170,25 @@ def _skip(self, word):
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = "alabaster"
+html_theme = "furo"
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
 #
+# See https://pradyunsg.me/furo/customisation/
+DATADOG_DARK_PURPLE = "#632CA6"
+
 html_theme_options = {
-    "description": "Datadog's Python APM client",
-    # https://alabaster.readthedocs.io/en/latest/customization.html#basics
-    "page_width": "1200px",
-    "sidebar_width": "400px",
+    "light_css_variables": {
+        "color-brand-primary": DATADOG_DARK_PURPLE,
+        "color-brand-content": DATADOG_DARK_PURPLE,
+    },
+    "dark_css_variables": {
+        # TODO: update to use a correct color, since dark purple is hard to read
+        # "color-brand-primary": DATADOG_DARK_PURPLE,
+        # "color-brand-content": DATADOG_DARK_PURPLE,
```

Comment:
`rgba(255, 255, 255, 0.76)` is the text color we use on app.datadoghq.com when in dark mode.

?

Reply:
I tried with the hex-equivalent of `#FFFFFFC2` and it looks not-great, as the text is already `#ffffffcc` and the links are semi-indistinguishable:

<img width="721" alt="Screenshot 2023-02-08 at 5 53 50 PM" src="https://user-images.githubusercontent.com/529516/217669810-205b9c8b-7b47-4392-9a6f-cd57b46ba5f7.png">


versus current:

<img width="886" alt="Screenshot 2023-02-08 at 5 54 04 PM" src="https://user-images.githubusercontent.com/529516/217669848-d95a77b5-74db-49ea-b86b-bee734399dbf.png">

Reply:
👍🏻 makes sense. happy to defer to a later date.